fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/book/yaml/SectionLoader.java

```diff
@@ -0,0 +1,85 @@
+package ch.bbw.fabbwled.lands.book.yaml;
+
+import ch.bbw.fabbwled.lands.book.SectionId;
+import ch.bbw.fabbwled.lands.exception.FabledTechnicalException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SectionLoader {
```

Comment:
```suggestion
public class YamlSectionLoader {
```
...to distinguish from the generic one?